packages/devextreme/js/__internal/ui/chat/messagelist.ts

```diff
@@ -417,6 +470,11 @@ class MessageList extends Widget<Properties> {
       case 'showDayHeaders':
         this._invalidate();
         break;
+      case 'typingUsers':
+        this._setIsReachedBottom();
```

Comment:
If I comment out lines 474 and 477, I don't see any failed QUnit tests. Perhaps I missed something. Could you please provide scenarios or test cases for what these lines are used?

Reply:
Thanks for attention, setTimeout is excessive [5b08da15e8a2606a141b186a140255003deb5016]